controls/os_spec.rb

```diff
@@ -228,3 +228,19 @@
     it { should be_empty }
   end
 end
+
+control 'os-10' do
+  impact 1.0
+  title "CIS: Disable unused filesystems"
+  desc "1.1.1 Ensure mounting of cramfs, freevxfs, jffs2, hfs, hfsplus, squashfs, udf, FAT"
+  describe file('/etc/modprobe.d/CIS.conf') do
```

Comment:
@chris-rock it disables the modules - from `man modprobe.d`:

```
install modulename command...
           This command instructs modprobe to run your command instead of
           inserting the module in the kernel as normal.
```

I think thats Okay for server systems. @atomic111 whats your opinion on this?

If we decide to accept this, we have to do the implementation in chef-os-hardening and ansible-os-hardening (cc @rndmh3ro) . In chef-os-hardening we have a global flag about desktop systems, so we could control a default based on this flag (and have an additional feature flag for a full control))

I'm not really happy with `CIS.conf` name, maybe we could use something like `hardeing.conf`, `dev-sec.conf`, maybe something else?